README.md

```diff
@@ -8,6 +8,54 @@ Orbit Spring Integration
 
 Integration for using the Spring Framework to manage an Orbit Cluster.
 
+## Orbit Dependency Injection Improvements
+
+With Orbit Spring, commonly accessed static methods and singletons in traditional Orbit can be injected as Spring beans.
+This makes classes easier to test.
+
+__Traditional Orbit:__
+
+```java
+public class GuestbookHandler
+{
+    public Task addEntry(String bookId, EntryDto entry)
+    {
+        Guestbook guestbook = Actor.getReference(Guestbook.class, bookId);
+        return guestbook.addEntry(entry);
+    }
+}
+```
+
+__With Orbit Spring:__
+
+```java
+public class GuestbookHandler
+{
+    @Autowired
+    private ActorService actorService;
+    
+    public Task addEntry(String bookId, EntryDto entry)
+    {
+        Guestbook guestbook = actorService.getReference(Guestbook.class, bookId);
```

Comment:
I don't have any strong preference. `ActorDirectory` seems like it may be ok. Alternatively if you want the service naming `ActorReferenceService` seems reasonable.